### ISSUE TITLE
Reserve all numbers 11-100 for metadata assignment (AssignmentObjectTypes)

### ISSUE THREAD

**Issue by iJungleboy** [2sic/2sxc#930]
Background: so far we're reserving 1-10 and we want to be sure to reserve all to 100. 

1. Please create a sql which adds "Reserved" entries for all numbers up to 100. 
2. Please include a condition, which only adds the reserved record IF it doesn't exist yet. This is to ensure that IF anybody had actually added some entries (incl. us - because for a while we accidentally used entries 11-13 for something) it won't kill that. 

So basically generate "autoid, Reserved, Reserved" for up to 100 rows.
